.gitignore

```diff
@@ -20,6 +20,8 @@
 .vagrant
 .noseids
 .ipynb_checkpoints
+.ipynb
+.csv
```

Comment:
I don't think we want to ignore these; pandas has many ipynb/csv files that are in the repo.

Reply:
Ok, removed